src/newrelic/app_test.go

```diff
@@ -488,7 +488,7 @@ func TestConnectPayloadEncoded(t *testing.T) {
 		`"metadata":{},` +
 		`"identifier":"one;two",` +
 		`"utilization":{"metadata_version":1,"logical_processors":22,"total_ram_mib":1000,"hostname":"some_host"},` +
-		`"event_harvest_config":{"report_period_ms":60000,"harvest_limits":{"error_event_data":100,"analytic_event_data":10000,"custom_event_data":10000,"span_event_data":1001}}` +
+		`"event_harvest_config":{"report_period_ms":60000,"harvest_limits":{"error_event_data":100,"analytic_event_data":10000,"custom_event_data":10000,"span_event_data":1001,"log_event_data":10000}}` +
```

Comment:
Like above I think setting `info.AgentEventLimits.SpanEventConfig.Limit` and checking the JSON would be a good idea.